Exiled.API/Features/Room.cs

```diff
@@ -146,15 +146,15 @@ public Color Color
         public IEnumerable<Camera> Cameras { get; private set; } = Enumerable.Empty<Camera>();
 
         /// <summary>
-        /// Gets or sets a value indicating whether or not the lights in this room are currently flickered on.
+        /// Gets or sets a value indicating whether or not the lights in this room are currently off.
         /// </summary>
-        public bool LightsOn
+        public bool LightsOff
```

Comment:
```suggestion
        public bool AreLightsOff
```
For consistency with other bools.

Reply:
Is not Are please :3

Reply:
> Is not Are please :3

???

Reply:
Ignore him he's a bit goofy with English